projects/subgraph-wells/subgraph.yaml

```diff
@@ -37,6 +39,10 @@ templates:
       abis:
         - name: Well
           file: ./abis/Well.json
+        - name: CurvePrice
+          file: ./abis/CurvePrice.json
+        - name: ERC20
+          file: ./abis/ERC20.json
       eventHandlers:
         - event: AddLiquidity(uint256[],uint256)
           handler: handleAddDeposit
```

Comment:
Shouldn't this be `handleAddLiquidity` ?

Reply:
Updated